pkg/builder/auth.go

```diff
@@ -102,9 +103,21 @@ func InPrivilegedUsersList(
 
 	// Users specified by Pod's environment variable "PRIVILEGED_USERS" are
 	// considered privileged.
-	c := pkgcfg.FromContext(ctx)
-	_, ok := pkgcfg.StringToSet(c.PrivilegedUsers)[userInfo.Username]
-	return ok
+	users := pkgcfg.StringToSlice(pkgcfg.FromContext(ctx).PrivilegedUsers)
```

Comment:
It would be beset if you could at the content of PR description here. Thanks

Reply:
This is no longer relevant I believe.